views/prisonerProfile/prisonerVisits/prisonerVisits.njk

```diff
@@ -44,10 +44,6 @@
     {
       text: visit.relationship,
       classes: 'govuk-table__cell--last' if visit.isLast 
-    },
-    {
-      html: hideIfNotFirst(visit.status, visit.isFirst),
-      classes: 'govuk-table__cell--last' if visit.isLast
```

Comment:
Remove the `status` column as it was incorrect.  Need to do more analysis to understand the visit data and then can add back in with better values e.g. only show `Attended` if visit is in the past and the visitor has been confirmed, show `Upcoming` for future visits etc.